ray/README.md

```diff
@@ -1,68 +1,59 @@
-# Deploying Ray with Open Data Hub
+# Deploying Ray on Open Data Hub via the KubeRay Operator
 
-Integration of [Ray](https://docs.ray.io/en/latest/index.html) with Open Data Hub on OpenShift. The ray operator and other components are based on https://docs.ray.io/en/releases-1.13.0/cluster/kubernetes.html
+The code found here is a subset of https://github.com/ray-project/kuberay. Specifically, the manifests needed to enable Ray with an Open Data Hub deployment. 
 
-## Components  of the Ray deployment
 
-1. [Ray operator](./operator/base/ray-operator-deployment.yaml): The operator will process RayCluster resources and schedule ray head and worker pods based on requirements. Image Containerfile can be found [here](https://github.com/thoth-station/ray-operator/blob/master/Containerfile). 
-2. [Ray CR](./operator/base/ray-custom-resources.yaml):  RayCluster Custom Resource (CR) describes the desired state of ray cluster.
-3. [Ray Cluster](./cluster/base/ray-cluster.yaml): Defines an instance of an example Ray Cluster. Image Containerfile can be found [here](https://github.com/thoth-station/ray-ml-worker/blob/master/Containerfile) 
- 
+## Components  of the KubeRay deployment
 
-## Deploy the RayCluster Components:
+1. Namespace 'ray-system'
+2. Custom Resource Definitions
+3. Operator
+4. Roles and Service Accounts
 
-Prerequisite to install RayCluster with ODH:
+## Deploy KubeRay:
+
+### Prerequisites to install KubeRay with ODH:
 
 * Cluster admin access
 * An ODH deployment
-* [Kustomize](https://kustomize.io/) 
-
-### Install Ray
-
-We will use [Kustomize](https://kustomize.io/) to deploy everything we need to use Ray with Open Data Hub. 
+* [Kustomize](https://kustomize.io/)
 
-#### Install the operator and custom resource 
 
-First use the `oc kustomize` command to generate a yaml containing all the requirements for the operator and the "raycluster" custom resource, then `oc apply` that yaml to deploy the operator to your cluster. 
+## Install KubeRay
```

Comment:
It occurs to me that these instructions bypass the ODH entirely. We're just manually installing kuberay from the raw kustomize artifacts. If we're providing a kfdef in this repo then IMO and stating that an ODH installation is a prerequisite then IMO our preferred instructions should be to apply that kfdef for installation.

Reply:
That is a good point @accorvin.  I can update the README so that is assumes you've installed KubeRay via the KfDef, then just how to confirms its installed and working. Would that be better?

Reply:
@MichaelClifford yeah, I like that plan.